app/views/users/posts/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="user-posts">
+  <% @user.posts.each do |post| %>
+    <div class="post-content">
+      <%= post.content %>
+      <div class="post-comments">
+        <% post.comments.each do |comment| %>
```

Comment:
- [ ] Kindly make use of the `recent_comments` method in the post model to return the most recent comments instead of returning all the comments for a post.

Reply:
Done!!
![image](https://github.com/lerfast/blog-app/assets/111823606/3560d0ff-9463-47ef-8ad3-87e448664417)